docs/admin-manual/file-and-cred-transfer.rst

```diff
@@ -531,8 +537,11 @@ in their job submit files.
 Using Vault as the OAuth client
 '''''''''''''''''''''''''''''''
 
-To instead configure HTCondor to use Vault as the OAuth client,
-install the ``condor-credmon-vault`` rpm.  Also install the htgettoken
+To configure HTCondor to use Vault as the exclusive oauth credential client,
+install the ``condor-credmon-vault`` rpm.
+Alternatively, to use Vault alongside the other credential clients (see note below),
+install the ``condor-credmon-multi`` rpm.
+Also install the htgettoken
```

Comment:
Also, reorder these detail sections to be consistently ordered with the list and detail paragraphs which precede them.

Consider rephrasing the section title as "Obtaining OAuth2 Tokens from Vault", and the first two sentences as something like [do we really not have .debs of these?  Why not?]:

```
If your submitters do not need to obtain tokens from the local issuer or via the native OAuth2 client, install the condor-credmon-vault package.

Otherwise, install the condor-credmon-multi package, which requires both the htgettoken package and additional configuration.
```